account_chart_update/wizard/wizard_chart_update.py

```diff
@@ -456,6 +456,8 @@ def diff_notes(self, template, real):
         if template._name == "account.tax.template":
             if not real.active:
                 result.append(_("Tax is disabled."))
+            if real.children_tax_ids.ids != template.children_tax_ids.ids:
```

Comment:
These IDs of course are not going to be the same, as one are taxes and the other are taxes templates. You have to use the same comparison as for parent taxes.